src/pathpyG/core/MultiOrderModel.py

```diff
@@ -248,7 +248,7 @@ def from_DAGs(
         dag_graph = next(iter(DataLoader(dag_data.dags, batch_size=len(dag_data.dags)))).to(config["torch"]["device"])
         edge_index = dag_graph.edge_index
         node_sequence = dag_graph.node_sequence
-        if dag_graph.edge_attr is None:
+        if dag_graph.edge_weight is None:
             edge_weight = torch.ones(edge_index.size(1), device=edge_index.device)
         else:
             edge_weight = dag_graph.edge_attr
```

Comment:
Shouldn't this also be called `dag_graph.edge_weight` instead of `dag_graph.edge_attr`?

Reply:
Yes, also that line needs to be modified. Thank you for noticing it!